includes/webform.report.inc

```diff
@@ -762,7 +762,7 @@ function webform_results_download_form_submit(&$form, &$form_state) {
  * Returns a temporary export filename.
  */
 function _webform_export_tempname() {
-  return backdrop_tempnam(config_get('webform.settings', 'webform_export_path'));
+  return backdrop_tempnam(config_get('webform.settings', 'webform_export_path'), 'webform_');
```

Comment:
No, it is indeed without the "e". https://api.backdropcms.org/api/backdrop/core%21includes%21file.inc/function/backdrop_tempnam/1